packages/helloworld/android/app/build.gradle

```diff
@@ -81,14 +81,14 @@ def enableProguardInReleaseBuilds = false
  * The preferred build flavor of JavaScriptCore (JSC)
  *
  * For example, to use the international variant, you can use:
- * `def jscFlavor = 'org.webkit:android-jsc-intl:+'`
+ * `def jscFlavor = "io.github.react-native-community:jsc-android-intl:2026004.+"`
```

Comment:
Q: Do we still need a JSC flavor? Can we just have a single string/flavor for it?

Reply:
just to keep it as backward compatible for a while. in the future we can remove this entirely

Reply:
Just a heads up: this requires a PR against https://github.com/react-native-community/template